src/utils/zulipVersion.js

```diff
@@ -112,13 +113,13 @@ export class ZulipVersion {
     }
 
     return result;
-  };
+  }
 
   /**
    * Compute a number[] to be used in .isAtLeast comparisons.
    */
-  _getComparisonArray = (raw: string): number[] => {
-    const { major, minor, patch, flag, numCommits } = this._getElements(raw);
+  _getComparisonArray(): number[] {
+    const { major, minor, patch, flag, numCommits } = this._getElements();
```

Comment:
Another thing I notice as I'm rereading this code with attention to how these initialization methods depend on each other: this ends up unnecessarily calling `_getElements` twice. Instead, this function could take a `VersionElements`, which the constructor can cheerfully pass it.